### DISCUSSION TITLE
Is single log out a thing in authenik? OIDC clients stay logged in after invalidation.

### DISCUSSION THREAD

**Discussion by hdlineage** [goauthentik/authentik#5666] [General]
I'm looking for a way to do single logout for all OIDC clients.
I was thinking maybe I can trigger the OIDC end session link in the log out flow but authentik does not appear to have the function.

As a SSO provider, you really should be able to control single sign out as well. Otherwise this poses a significant security risk.

**Comment by tedstriker** [comment]
It seems like there is a standard called backchannel logout ([docs](https://openid.net/specs/openid-connect-backchannel-1_0.html)), but I couldn't find anything about it in the Authentik documentation.
Is this actually implemented?
